app/assets/javascripts/discourse/app/controllers/composer.js

```diff
@@ -770,14 +770,16 @@ export default Controller.extend({
 
     // TODO: This should not happen in model
     const imageSizes = {};
-    $("#reply-control .d-editor-preview img").each((i, e) => {
-      const $img = $(e);
-      const src = $img.prop("src");
-
-      if (src && src.length) {
-        imageSizes[src] = { width: $img.width(), height: $img.height() };
-      }
-    });
+    $("#reply-control .d-editor-preview img")
+      .not(".onebox.allowlistedgeneric img")
```

Comment:
Can we try to avoid jQuery while we're at it? (As mentioned in chat, and as the comment in line 771 says, ideally we wouldn't do this here at all, but in the meantime we can at least reduce the jQuery footprint.)

Also, I don't think it would hurt to apply this to all images inside oneboxes (i.e. not just `allowlistedgeneric` ones).